pages/common/git-commit.md

```diff
@@ -14,3 +14,7 @@
 - Replace the last commit with currently staged changes:
 
 `git commit --amend`
+
+- Commit only specific files or folders:
+
+`git commit -m {{message}} {{path/to/my/file}} {{path/to/my/second_file}}`
```

Comment:
I would remove the `-m`. It is already specified above and doesn't need to be used again. This example is clearer without it.